model/integration_action_test.go

```diff
@@ -161,3 +161,27 @@ func TestSubmitDialogResponseToJson(t *testing.T) {
 		assert.Nil(t, r)
 	})
 }
+
+func TestPostActionIntegrationEquals(t *testing.T) {
```

Comment:
I'd suggest to also add a comparable type case and one where the contents do not match.